routers/api/v1/api.go

```diff
@@ -835,7 +835,8 @@ func Routes(sessioner func(http.Handler) http.Handler) *web.Route {
 					})
 					m.Group("/{index}", func() {
 						m.Combo("").Get(repo.GetIssue).
-							Patch(reqToken(), bind(api.EditIssueOption{}), repo.EditIssue)
+							Patch(reqToken(), bind(api.EditIssueOption{}), repo.EditIssue).
+							Delete(reqToken(), repo.DeleteIssue)
```

Comment:
```suggestion
							Delete(reqToken(), reqAdmin(), repo.DeleteIssue)
```

Reply:
If I intend to extend the behaviour (do this via the Web UI), would it reuse this route, or, would there be a similar feature in other routes? I suppose so?

What about also allowing admins of the repo to permanently delete issues? Does that make sense?

Reply:
> If I intend to extend the behaviour (do this via the Web UI), would it reuse this route, or, would there be a similar feature in other routes? I suppose so?

The current code lives in the api part, so yo would need to move the function to a more generic package(`models`), but most if not all of those functions always expects the code on web/api to handle the correct authentication etc.

> What about also allowing admins of the repo to permanently delete issues? Does that make sense?

see the `reqAdmin` function's code, it allows the site's admins + repo's admins.

Reply:
Ah okay. Indeed, I used the Repo.IsAdmin, I missed this. Considered to restrict it to site-admins at first but apparently did otherwise. Thank you for the clarification.